lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,5 @@
-
 def palindrome_permutation?(string)
-  
+  h = Hash.new { |h, k| h[k] = 0 }
+  string.each_char { |c| h[c] += 1 }
+  h.values.select(&:odd?).count <= 1
```

Comment:
A space between each of these three lines would be helpful to visually separate variable initialization, intermediate data structure preparation, and computing the final result.